### DISCUSSION TITLE
手机端侧边栏点开后无法关闭

### DISCUSSION THREAD

**Discussion by Mitpoppy** [Qexo/Qexo#389] [Ideas]
点开后点哪里都关不了，点了侧边栏的按钮跳到别的页面才能关闭，我感觉是个bug吧，比较影响操作

**Comment by Fgaoxing** [comment]
e,老问题了



在 2023-05-25 19:45:16，"Mitpoppy" ***@***.***> 写道：

点开后点哪里都关不了，点了侧边栏的按钮跳到别的页面才能关闭，我感觉是个bug吧，比较影响操作
